src/pages/interviews.tsx

```diff
@@ -232,9 +232,7 @@ function Applicant({ type, applicant, interviews, refetchInterviews,
 
       {/* 生源 */}
       <TdEditLink>
```

Comment:
Remove `<TdEditLink>`. It's okay if we lose click-to-edit function in this cell.